### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by xiaolou86**

### CODE REVIEW

**Review by louis-md — APPROVED** [review]

File: safe-apps/release.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ Remember that **CORS** needs to be configured correctly on the `manifest.json` s
...
+When a user opens the app, it should automatically select the Safe as a wallet. Ensure to check the case if the user previously opened the app outside the Safe with another wallet.
 
 ### The Safe team has reviewed the Safe App
 
 The requirement doesn't apply for battle-tested applications hosted on the same domain as the main dapp.
 
 While we won't be able to do a proper audit for your Safe App, we still would like to look at the source code to raise issues or suggest improvements. Whether your Safe App is open or closed source, please send us either a link to the public repository or an invitation to the private repository.
 
-We also would like to make a rough functional review of the App, so please provide us with a high-level test plan/feature list that allows our QA team to ensure everything works as intended in production. Video walkthroughs are also welcome.
+We also would like to make a rough functional review of the App, so please provide us with a high-level test plan/feature list that allows our QA team to ensure everything works as intended in production. Video walkthroughes are also welcome.
```
Comment (germartinez) [inline safe-apps/release.md:49]:
```suggestion
We also would like to make a rough functional review of the App, so please provide us with a high-level test plan/feature list that allows our QA team to ensure everything works as intended in production. Video walkthroughs are also welcome.
```

File: safe-core-api/rpc-requirements.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ For the L2 version of the Safe, no special RPC methods are used. The most demand
 That's not written in stone. Tx service has some environment variables that can be configured to set a limit on the number of blocks that are processed together (`ETH_EVENTS_BLOCK_PROCESS_LIMIT_MAX`), but the default behaviour is trying to detect the best configuration for every network similar to how [TCP congestion control](https://en.wikipedia.org/wiki/TCP_congestion_control) works.Indexer tries to process a low number of blocks (currently 50). Depending on that:
 
 * If request takes **less than 1 second**, node can process more. Number of blocks to fetch is duplicated for the next request.
-* If request takes **less than 3 seconds**, number of blocks to process are incremented by an small amount (currently 20).
-* If request takes **more than 20 seconds**, number of blocks to process are decremented by an small amount (currently 20).
+* If request takes **less than 3 seconds**, number of blocks to process are incremented by a small amount (currently 20).
```
Comment (germartinez) [inline safe-core-api/rpc-requirements.md:38]:
```suggestion
* If the request takes **less than 3 seconds**, the number of blocks to process is incremented by a small amount (currently 20).
```

File: safe-core-api/rpc-requirements.md [hunk 3]
```diff
@@ -35,8 +35,8 @@ For the L2 version of the Safe, no special RPC methods are used. The most demand
 That's not written in stone. Tx service has some environment variables that can be configured to set a limit on the number of blocks that are processed together (`ETH_EVENTS_BLOCK_PROCESS_LIMIT_MAX`), but the default behaviour is trying to detect the best configuration for every network similar to how [TCP congestion control](https://en.wikipedia.org/wiki/TCP_congestion_control) works.Indexer tries to process a low number of blocks (currently 50). Depending on that:
 
 * If request takes **less than 1 second**, node can process more. Number of blocks to fetch is duplicated for the next request.
```
Comment (germartinez) [inline safe-core-api/rpc-requirements.md:37]:
```suggestion
* If the request takes **less than 1 second**, the node can process more. The number of blocks to fetch is duplicated for the next request.
```

File: safe-core-api/rpc-requirements.md [hunk 4]
```diff
@@ -35,8 +35,8 @@ For the L2 version of the Safe, no special RPC methods are used. The most demand
...
-* If request takes **less than 3 seconds**, number of blocks to process are incremented by an small amount (currently 20).
-* If request takes **more than 20 seconds**, number of blocks to process are decremented by an small amount (currently 20).
+* If request takes **less than 3 seconds**, number of blocks to process are incremented by a small amount (currently 20).
+* If request takes **more than 20 seconds**, number of blocks to process are decremented by a small amount (currently 20).
```
Comment (germartinez) [inline safe-core-api/rpc-requirements.md:39]:
```suggestion
* If the request takes **more than 20 seconds**, the number of blocks to process is decremented by a small amount (currently 20).
```

File: safe-core-api/rpc-requirements.md [hunk 5]
```diff
@@ -35,8 +35,8 @@ For the L2 version of the Safe, no special RPC methods are used. The most demand
...
 * If request takes **more than 30 seconds**, number of blocks to process are halved.
```
Comment (germartinez) [inline safe-core-api/rpc-requirements.md:40]:
```suggestion
* If the request takes **more than 30 seconds**, the number of blocks to process is halved.
```

File: safe-apps/release.md [hunk 6]
```diff
@@ -38,15 +38,15 @@ Remember that **CORS** needs to be configured correctly on the `manifest.json` s
 
 ### The app auto connects to the Safe
 
-When a user opens the app, it should automatically select the Safe as a wallet. Ensure to check the case if the user previously opened the app outside of the Safe with another wallet.
+When a user opens the app, it should automatically select the Safe as a wallet. Ensure to check the case if the user previously opened the app outside the Safe with another wallet.
```
Comment (germartinez) [inline safe-apps/release.md:41]:
```suggestion
When a user opens the app, it should automatically select the Safe as a wallet. Ensure to check the case if the user previously opened the app outside of the Safe with another wallet.
```

**Review by germartinez — CHANGES_REQUESTED** [review]

**Comment by xiaolou86** [comment]
@germartinez Thanks for your review.

**Pull request merged**